community-tutorials/debootstrap/01-en.md

```diff
@@ -0,0 +1,311 @@
+
+For this tutorial a terminal access, like provided in the Netcup SCP, is required. If your hoster does not offer such a feature, rescaling your boot partition is much more complicated. Please also checkout Reference 2 for rebooting your old system on failure.
+
+## References
+
+This tutorial was heavily influenced by:
+- https://www.debian.org/releases/stretch/amd64/apds03.html.en (Ref1)
+- https://gist.github.com/varqox/42e213b6b2dde2b636ef (Ref2)
+
+## Consider LVM
+
+In this tutorial, we will create all partitions without LVM. This again results in a setup, where it is hard to scale partitions outside of their ordering, meaning each partition can only be scaled "to the right" and free space that is not right of a partition can not be used. LVM circumvents this problem at the cost of some performance. If there is any reliable benchmark out there for LVM impact on netcup vServers please let me know.
+
+## Consider LUKS
+
+Like with any physical or virtual drive, an unencrypted drive can be mounted into any other system and is available just like you would have it inside your vServer, meaning all data can be read. The only countermeasure is to encrypt your harddrive with luks (or any other encrypting mechanism). This is also not part of this tutorial. If you do encrypt your partitions, you will have to have an extra partition for `/boot` outside of your encrypted area to enable grub to boot your system. There are plenty of other options out there to make this work. As long as you do not touch your original system, this tutorial is the perfect opportunity to play with encryption.
+
+# Step 1 - (Making Space)
+Since we are going to setup the new OS parallel to the existing one, we need space. If there already exists an empty partition on your system you can skip this step.
+
+**Working with partitions can be quite dangerous. Please be careful.**
+
+If you haven't worried too much about your vServer setup, there are good chances your layout might look something like this:
+
+```
+|start|[sda1 - grub][sda2 - swap][sda3 - ext4 - mounted as / - ]|end|
+```
+
+This setup is quite disadvantageous since we are not able to shrink an ext4 filesystem while it is mounted. With Netcup we are lucky since inside the Server Control Panel, we can boot from an iso **and** have a console with mouse support.
+
+*For this step we will have some downtime. For my VPS 16years (with an HDD) downsizing the ext4 by around 500 GB took about 50 minutes.*
+
+## Step 1.1 - (Shrink existing partition)
+![Gparted DVD in Netcup SCP](images/netcup_scp.png)
+
+Log into your SCP, select your server, then go to Media and select DVD Drive. There we insert the GParted Live CD and set the server to boot from DVD via switch. Afterwards we reboot the server and open the console via the >_Screen menupoint.
```

Comment:
Could you please add where to change the boot order.

Reply:
Added a screenshot that might clarify